libportal/portal-private.h

```diff
@@ -71,3 +73,30 @@ const char * portal_get_bus_name (void);
 #define FLATPAK_PORTAL_BUS_NAME "org.freedesktop.portal.Flatpak"
 #define FLATPAK_PORTAL_OBJECT_PATH "/org/freedesktop/portal/Flatpak"
 #define FLATPAK_PORTAL_INTERFACE "org.freedesktop.portal.Flatpak"
+
+static inline int
```

Comment:
What is the difference with g_autofd? it would be nice if the commit explains why.

Reply:
moved the first two commits to a different MR since it's not relevant here anymore https://github.com/flatpak/libportal/pull/172

Reply:
added a comment about the reason why we can't use g_autofd to the other mr